any.hpp

```diff
@@ -331,20 +331,21 @@ struct static_any
 		if (another.function_ == nullptr)
 			return;
 
+		static_any temp = *this;
```

Comment:
This will call 'copy_or_move_from_another', which has the same borked exception handling.